src/data/scene.ts

```diff
@@ -50,7 +50,9 @@ export interface SceneConfig {
 }
 
 export interface SceneEntities {
-  [entityId: string]: string | { state: string; [key: string]: any };
+  [entityId: string]:
+    | string
+    | { state: string; [key: string]: any; entity_only?: boolean | undefined };
```

Comment:
Should we tuck the `entity_only` flag away in a `metadata` object?

Reply:
Do we even need `entity_only`? If we allow this, we would just make it so that picking an entity in the entity list doesn't automatically upgrade to the device.

Reply:
So you pick a device for devices, you pick an entity for entities.

Reply:
The problem is that the scene config does not have a concept of devices which are included, frontend derives the included devices by looking up device ids for entities included in the scene.

It means we need some way for frontend to keep track of which devices should be included by storing additional data in the scene config. The `entity_only` is an opt-out flag; it would maybe be more logical to make it an opt-in flag instead?

Reply:
If it's for the frontend only, we generally use `metadata`.